src/CanvasContainer.tsx

```diff
@@ -204,7 +204,11 @@ export const CanvasContainer: React.FC = ({ children }) => {
           );
         }
       } else if (!e.metaKey && !e.ctrlKey) {
-        canvasService.send(canvasModel.events.PAN(e.deltaX, e.deltaY));
+        if (isMac() || !e.shiftKey) {
+          canvasService.send(canvasModel.events.PAN(e.deltaX, e.deltaY));
+        } else {
+          canvasService.send(canvasModel.events.PAN(e.deltaY, 0));
```

Comment:
After playing with this a little bit more I'm unsure if this is the best fix for the issue. It works for this specific case but I'm hesitant to say that it's universally good. It would be quite good if I could only apply this for mouse interactions but as far as I can tell they are indistinguishable from touchpad interactions.